### ISSUE TITLE
Rule D413 produces a false positive for a section headed `Examples`.

### ISSUE THREAD

**Issue by JDawson-Camlin** [astral-sh/ruff#6037]
The bug is exhibited by the command
```shell
ruff --isolated --select D413 /path/to/file.py
```

If `file.py` is
```python
"""Title.

Examples
--------
Body.
"""
```
there is a false positive:
```
file.py:1:1: D413 [*] Missing blank line after last section ("Examples")
Found 1 error.
[*] 1 potentially fixable with the --fix option.
```

However, there is, correctly, no error detected if `file.py` is
```python
"""Title.

Heading
-------
Body.
"""
```

Running `ruff --version` prints `ruff 0.0.280`.

**Comment by charliermarsh** [comment]
So I _think_ this is correct. This rule wants you to format like:

```py
"""Title.

Examples
--------
Body.

"""
```

Which is consistent with pydocstyle's own implementation of this rule.

If you're looking to guard against:

```py
"""Title.

Examples
--------
Body."""
```

That's instead covered by D209.

Note that this is kind of a strange rule -- it's not included in the NumPy, Google, or PEP 257 conventions.

**Comment by charliermarsh** [comment]
Thanks, I think that documentation is just wrong. I'll fix it.

I believe that "Heading" is not a recognized section header (unlike "Examples"), `pydocstyle` doesn't error there either.

**Comment by JDawson-Camlin** [comment]
What headings are recognized?

**Comment by charliermarsh** [comment]
```python
[
    "Args",
    "Arguments",
    "Attention",
    "Attributes",
    "Caution",
    "Danger",
    "Error",
    "Example",
    "Examples",
    "Extended Summary",
    "Hint",
    "Important",
    "Keyword Args",
    "Keyword Arguments",
    "Methods",
    "Note",
    "Notes",
    "Other Args",
    "Other Arguments",
    "Other Params",
    "Other Parameters",
    "Parameters",
    "Raises",
    "References",
    "Return",
    "Returns",
    "See Also",
    "Short Summary",
    "Tip",
    "Todo",
    "Warning",
    "Warnings",
    "Warns",
    "Yield",
    "Yields",
]
```

**Comment by JDawson-Camlin** [comment]
Thank you.

**Comment by charliermarsh** [comment]
I can add something to the FAQ around this point.
